src/components/molecules/Common/AssetModal/AssetContainer/hooks.ts

```diff
@@ -6,6 +6,10 @@ export type SortType = "date" | "name" | "size";
 
 export type LayoutTypes = "medium" | "small" | "list";
 
+export const fileFormats = ".kml,.czml,.json,.gltf,.glb";
```

Comment:
```suggestion
export const fileFormats = ".kml,.czml,.topojson,.geojson,.json,.gltf,.glb";
```

Reply:
Ah I had those but when I googled those formats it seemed like they were just json files with specific formatting. I'll re-add those!